### ISSUE TITLE
subscription management app

### ISSUE THREAD

**Issue by owocki** [gitcoinco/skunkworks#115]
what this would do:
scans your ETH account to see all 1337 compatible subscriptions you haave.. and letes you toggle them on and off

**Comment by owocki** [comment]
@androolloyd donno if anything like this is on groundhogs roadmap at all..

**Comment by androolloyd** [comment]
Yeah we’re working on one that works for our version. Should be released shortly. Scanning on chain data will require us to converge on event types, given that we’re going on to different directions currently.
